README.md

```diff
@@ -6,33 +6,66 @@
 [![Discourse status](https://img.shields.io/discourse/https/meta.discourse.org/status.svg?style=for-the-badge)](https://forums.habitat.sh)
 [![Open Source Helpers](https://www.codetriage.com/habitat-sh/habitat/badges/users.svg)](https://www.codetriage.com/habitat-sh/habitat)
 
-Want to try Habitat? [Get started here](https://www.habitat.sh/learn/).
+<img src="https://github.com/habitat-sh/habitat/blob/master/www/source/images/habitat-logo.png" width="150">
```

Comment:
I'm not sure we need the logo here, since we have the heading up at the top.

Reply:
I think this could be a convenient way for anyone who wants to grab and use the Habitat logo, and it links to the logo file directly. Also, it is the only place in the README that introduces the logo mark, not just the header that enters its name.